.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
     - sudo apt-get install -qq libboost-python1.53-dev libgoogle-perftools-dev libgmp-dev libboost1.53-dev git cmake
 
     - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 80 --slave /usr/bin/g++ g++ /usr/bin/g++-4.8
+    - sudo apt-get install gfortran libopenblas-dev liblapack-dev
```

Comment:
If I correctly understand, numpy compile some fortran code when it is installed with pip install numpy.